ruma-federation-api/src/membership/create_join_event/v2.rs

```diff
@@ -31,7 +31,6 @@ ruma_api! {
 
     response: {
         /// Full state of the room.
-        #[serde(with = "crate::serde::room_state")]
         pub room_state: RoomState,
     }
 }
```

Comment:
All files should have a final newline. To make your editor do this automatically, you might want to install an [editorconfig](https://editorconfig.org/) plugin.